recipes/statistics/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "statistics" %}
+{% set version = "1.0.3.5" %}
+{% set sha256 = "2dc379b80b07bf2ddd5488cad06b2b9531da4dd31edb04dc9ec0dc226486c138" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - statistics
+
+about:
+  home: https://pypi.python.org/pypi/statistics
+  license: Apache-2.0
+  license_family: Apache
```

Comment:
Please follow-up on this [request]( https://github.com/digitalemagine/py-statistics/issues/2#issuecomment-285830188 ) to get the license file included in the `sdist` on PyPI. On the next release when the license file is included, please add `license_file: LICENSE` to this section. Thanks.